### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Java_Project Management_Usability : Headings on the page are programmatically marked as headings along with previous paragraph content.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/vscode-docs#6314]
### GitHub Tags
#A11yUsable; #A11ySev4; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023;

### Environment Details
Application Name: Visual Studio Code Services
URL: [https://code.visualstudio.com/](https://code.visualstudio.com/)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 113.0.5672.93 (Official Build) (64-bit)

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Tab till "Docs" link and press enter.
3. Tab till "JAVA" link in the left navigation and press enter.
4. Tab till "Project Management" link which is present under the java and press enter.
5. Inspect the page headings.
6. Observe that  Headings on the page are programmatically marked as headings along with previous paragraph content.

### Actual Result
Headings on the page are programmatically marked as headings along with previous paragraph content.
Note:
Similar issue is observed throughout the application.

### Expected Result
Only the Visually appearing headings on the page should be marked as headings programmatically and not along with previous paragraph content.

### User Impact
Users will face difficulties to understand the structural view of the page if in heading markup other content is also included.

### Attachment:

![Headings on the page are programmatically marked as headings along with previous paragraph content](https://github.com/microsoft/vscode-docs/assets/91586101/9c4d7e0e-c713-404f-b2a5-40e59126cbdd)

**Comment by rzhao271** [comment]
The headings are spaced so that the right-side navigation and permalinks can correctly reveal the headings while scrolling to them.

I'm wondering how the spacing could affect screenreader users? The markup itself is proper, and I don't believe that a screenreader would report on the spacing of the headers.
